code/data_structures/src/tree/binary_tree/binary_tree/traversal/zigzag/zigzag.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <stack>
+
+class Node
+{
+    public:
+        int info;
+        Node* left_child;
+        Node* right_child;
+        Node (int info)
+        {
+            this->info = info;
+            left_child = nullptr;
+            right_child = nullptr;
+        }
+};
+
+class BinaryTree
+{
+    public:
+        Node* root;
+        BinaryTree()
+        {
+            root = nullptr;
```

Comment:
Should use member initialization list also.

Reply:
Actually, yes, you should use member initializer lists, as that is the general recommended practice.